CHANGELOG.md

```diff
@@ -2,18 +2,20 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [0.37.0]()
+## [0.37.0](https://github.com/AliceO2Group/Bookkeeping/releases/tag/%40aliceo2%2Fbookkeeping%400.37.0)
```

Comment:
The changelog need to target 0.38.0 I think

Reply:
Done